Applications/Piano/RollWidget.h

```diff
@@ -46,8 +46,8 @@ class RollWidget final : public GUI::ScrollableWidget {
 
     TrackManager& m_track_manager;
 
-    int m_roll_width;
-    int m_num_notes;
-    double m_note_width;
+    int m_roll_width { 0 };
+    int m_num_notes { 0 };
+    double m_note_width { 0.0 };
```

Comment:
This is embarrassing, how come we didn't catch it during review?

Reply:
These are basically harmless since they get assigned on first paint event anyway.
But still, should have been caught!